packages/apsara-ui/src/TableV2/VirtualisedTable.tsx

```diff
@@ -37,8 +37,8 @@ function VirtualisedTable({
     loading = false,
 }: ITableProps) {
     useEffect(() => {
-        if (!dataFetchFunction && items?.length) {
-            setData({ rows: items });
+        if (!dataFetchFunction) {
```

Comment:
what does `dataFetchFunction` for?

Reply:
If I remember correctly it was to implement server side pagination, so we had option to pass data directly or use a function to fetch data.